Examples.md

```diff
@@ -3,3 +3,21 @@ You want to see the examples? Did you even read all the features of DreamBerd? O
 
 Please go back to the features page by following [this link](https://github.com/TodePond/DreamBerd/blob/main/README.md).<br>
 When you have finished reading the features, you can find the _real_ examples [here](https://github.com/TodePond/DreamBerd/blob/main/res/Examples.md).
+
+```DREAMBERD PUBLIC LICENSE
```

Comment:
Please mispell DREAMBERD here, otherwise it implies the license isn't owned by the Dreamberd foundation